### ISSUE TITLE
Sonar board inclusion?

### ISSUE THREAD

**Issue by anfederman** [UbiquityRobotics/breadcrumb_learn#19]
The build instructions say remove the sonar board, but don't say to put it back in if there is no Lidar.
Also if the sonar board is to be included, it should be reinstalled after the pi is already installed.
I discovered the sonar board will prevent the standard LS-n301 from being mounted lower left.

I am not sure how we want to handle this, but I suggest addressing it in some way in the assembly instructions.

**Comment by JanezCim** [comment]
Yes, currently if the lidar is mounted left or right, its big connector is in the way of mounting sonars. Solution would be to mount it on the tower or on the front plate in case you want both. We are now also testing some new sonar layouts and we have yet to see if the sonars will be able to replace the lidar entirely. If not, we are looking into mounting a different lidar, which has a smaller connector (https://www.amazon.com/innomaker-Omni-Directional-Scanning-Resistance-Ranging/dp/B08GJJX41D/), and then both sonars and lidar will be able to be mounted. 

So the answer is, we include the sonar board into instructions once we confirm it works.

**Comment by anfederman** [comment]
I was able to get both the sonar and lidar mounted at the same time.

**Comment by JanezCim** [comment]
aha okay, thats good news. You mounted then without twisting the sonar PCB entirely? 

So the new sonar layouts are being tested by @GreTimotej in https://github.com/UbiquityRobotics/breadcrumb/issues/269 so as soon as he confirms how sonars are working, we can decide if we include them into tutorials

**Comment by anfederman** [comment]
Yeah check out the video posted in chat.

**Comment by JanezCim** [comment]
The new sonar board design will not clash with camera
